### PR TITLE
Fix typos and indents in `Operator`/`Install` YAML

### PR DESCRIPTION
**Pull request by tlwu2013**
Fixed a few typos in `Operator` object YAML and indents in `Install` object YAML.

### CODE REVIEW

**Comment by tlwu2013** [comment]
FYI, @njhale

**Comment by russellb** [comment]
This looks like a simple enhancement tweak that fell through the cracks.  I've pinged the OLM dev team to verify the changes are correct so we can merge this.

File: enhancements/olm/simplify-apis.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ status:
   components:
    matchLabels:
       operators.coreos.com/operator:v2alpha1/plumbus: ""
-   refs:
+   resources:
```
Comment (njhale) [inline enhancements/olm/simplify-apis.md:132]: it's actually `refs`, see https://github.com/operator-framework/api/blob/e29d40c2eb3eab6a2789a671886b8966ce8e3c0a/pkg/operators/v1/operator_types.go#L47

File: enhancements/olm/simplify-apis.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ status:
 
 **Note:** *Both namespace and cluster scoped resources are gathered using this label selector. Namespace scoped components are selected across all namespaces.*
 
-Once associated with an `Operator`, a component's reference will be listed in the `status.components.resource` field of that `Operator`. Component references will also be enriched with abnormal status conditions relevant to the operator. These conditions should follow [k8s status condition conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) and in some cases may be copied directly from the component status.
+Once associated with an `Operator`, a component's reference will be listed in the `status.components.resources` field of that `Operator`. Component references will also be enriched with abnormal status conditions relevant to the operator. These conditions should follow [k8s status condition conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) and in some cases may be copied directly from the component status.
```
Comment (njhale) [inline enhancements/olm/simplify-apis.md:285]: `status.components.refs`

File: enhancements/olm/simplify-apis.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ status:
   components:
    matchLabels:
       operators.coreos.com/operator:v2alpha1/plumbus: ""
```
Comment (njhale) [inline enhancements/olm/simplify-apis.md:131]: could you also correct this label convention? it's `operators.coreos.com/<operator-name>: ""` now. So in this case, we'd have `operators.coreos.com/plumbus`.

**Comment by njhale** [comment]
@tlwu2013, thank you for updating this and sorry for the long wait. I think that after this change we'll want to move the enhancement over to the [operator-framework org](https://github.com/operator-framework/enhancements).

**Pull request closed without merging**